backend/lucene/src/main/java/org/hibernate/search/backend/lucene/search/projection/impl/DistanceFieldSearchProjectionBuilderImpl.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.lucene.search.projection.impl;
+
+import org.hibernate.search.backend.lucene.document.model.impl.LuceneIndexSchemaFieldNode;
+import org.hibernate.search.backend.lucene.search.impl.LuceneSearchTargetModel;
+import org.hibernate.search.engine.search.projection.spi.DistanceFieldSearchProjectionBuilder;
+import org.hibernate.search.engine.spatial.DistanceUnit;
+import org.hibernate.search.engine.spatial.GeoPoint;
+
+public class DistanceFieldSearchProjectionBuilderImpl extends AbstractFieldSearchProjectionBuilderImpl<GeoPoint, Double>
```

Comment:
Same comment as on the Elasticsearch side: it is probably not a good idea to make the distance projection builder extend that class.